### ISSUE TITLE
[dev.icinga.com #7984] Use semantic markup whenever color cues are used

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icingaweb2#1248]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/7984

**Created by mhein on 2014-12-03 13:37:12 +00:00**

Assignee: _(none)_
Status: _Closed_  (closed on _2016-02-19 16:07:12 +00:00_)
Target Version: _Accessibility_
Last Update: _2016-02-19 16:07:12 +00:00 (in Redmine)_

---

http://www.w3.org/TR/2014/NOTE-WCAG20-TECHS-20140916/G138 wrote:

> The objective of this technique is to combine color and semantic markup to convey information. Most users can quickly scan the content to locate information conveyed by using color. For users who cannot see color, semantic markup can provide a different type of cue. User agents can then make this type of structure perceivable to the user, for example using a different visual presentation for different types of structures or by using a different voice or pitch in an auditory presentation. \[...\]

**Acceptance Criteria:**

-   Check that the same information is available through semantic markup.
